lib/tlaw/has_parent.rb

```diff
@@ -0,0 +1,19 @@
+module TLAW
+  module HasParent
+    attr_reader :parent
+
+    # Returns [parent, parent.parent, ...]
+    def parents
+      result = []
+      cursor = self
+      while (cursor = cursor.parent)
+        result << cursor
+      end
+      result
+    end
+
+    def lineage
```

Comment:
I am not sure about the name, never met this word used in this context :)
Are there any other gems/Ruby project that provide the same API ("me and my parents") you aware of?.. What names do they prefer?

Reply:
Well, there's `Module#ancestors`, which includes the module itself... but we can't use that. It's not really mandatory. I've removed it from the PR, we can add it later if need be.